db.js

```diff
@@ -3355,167 +3431,234 @@ module.exports.CreateDB = function (parent, func) {
     }
 
     // Perform a server backup
-    obj.performingBackup = false;
```

Comment:
why has this been removed? as it should default to false otherwises when it sets later the value is undefined

Reply:
I just moved it to the top of the module for readability, so it's still defined and initialized to false. But that's just part of my defaults, vars at the top. Maybe shouldn't have done that.

Reply:
oh sorry yes you are correct!
https://github.com/PTR-inc/MeshCentral/blob/adcb2ca8eae2394c61f5bdeaf25260cb80017f10/db.js#L42
thats ok and yeh its better at the beginning
because then in other places of the code we can check if its doing backups or not etc 👍